content/en/database_monitoring/setup_sql_server/troubleshooting.md

```diff
@@ -41,7 +41,9 @@ If the `datadog` user is unable to log into the SQL Server instance, please ensu
 
 Microsoft also provides a helpful doc on troubleshooting these types of errors, which can be [followed here][5].
 
-### SQL Server Unable to connect due to “Invalid connection string attribute” {#tcp-connection-error}
+### SQL Server Unable to connect {#tcp-connection-error}
```

Comment:
```suggestion
### SQL Server unable to connect '{#tcp-connection-error}'
```

Reply:
Without the apostrophes, the text doesn't show up in the doc.

Reply:
why capitalize unable, but leave "connect" lowercase?

> Without the apostrophes, the text doesn't show up in the doc.

this text is not meant to show up in the doc. It's a permanent link, which is referenced in our agent. The purpose is that if we want to change the heading it won't break the agent.

Reply:
Ah okay, I took out the apostrophes. 

My change makes "Unable" lowercase, like "connect", so that the heading is sentence case.

Reply:
yeah sorry :face-palm: i read it backwards